N21: Add chip-8-rs (#608)
rust-gamedev/rust-gamedev.github.io: content/news/021/index.md

[bevy_webgl2]: https://github.com/mrk-its/bevy_webgl2
[version 0.4]: https://github.com/mvlabat/bevy_egui/blob/main/CHANGELOG.md

### [chip-8-rs]

![Screenshot of Pong with debugger](chip-8-rs.jpg)
_Debugging Pong_

> Chip-8-rs (https://github.com/JonathanMurray/chip-8-rs) by @jonathanmurray is

Reviewer: ```suggestion
[Chip-8-rs][chip-8-rs] by @jonathanmurray is
```

a _CHIP-8_ emulator with some basic debugging functionality.

When running a game through the emulator, CHIP-8 instructions are listed
next to the main display, with the currently executed one highlighted. By
running at a very low clock-frequency (and pausing/resuming) you can step
through a program one instruction at a time, to better understand how it
works (or doesn't work!).

See it in action here: https://youtu.be/nVDJ5PZpPfI?t=72

> 

Reviewer: ```suggestion

[chip-8-rs]: https://github.com/JonathanMurray/chip-8-rs
```

[bevy_webgl2]: https://github.com/mrk-its/bevy_webgl2
[version 0.4]: https://github.com/mvlabat/bevy_egui/blob/main/CHANGELOG.md

### [chip-8-rs]

![Screenshot of Pong with debugger](chip-8-rs.jpg)
_Debugging Pong_

[Chip-8-rs][chip-8-rs] by @jonathanmurray is
a _CHIP-8_ emulator with some basic debugging functionality.

When running a game through the emulator, CHIP-8 instructions are listed
next to the main display, with the currently executed one highlighted. By
running at a very low clock-frequency (and pausing/resuming) you can step
through a program one instruction at a time, to better understand how it
works (or doesn't work!).

> See it in action here: https://youtu.be/nVDJ5PZpPfI?t=72

Reviewer: Oops, the linter doesn't like bare URLs...
```suggestion
See it in action on [YouTube][chip-8-rs-video].
```

> [chip-8-rs]: https://github.com/JonathanMurray/chip-8-rs

Reviewer: ```suggestion
[chip-8-rs]: https://github.com/JonathanMurray/chip-8-rs
[chip-8-rs-video]: https://youtu.be/nVDJ5PZpPfI?t=72
```
